test/e2e/history_test.go

```diff
@@ -44,6 +44,13 @@ var _ = Describe("Podman history", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 		Expect(session.OutputToStringArray()).ToNot(BeEmpty())
+
+		session = podmanTest.Podman([]string{"history", "--format", "{{.CreatedAt}};{{.Size}}", ALPINE})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+		lines := session.OutputToStringArray()
+		Expect(lines).ToNot(BeEmpty())
+		Expect(lines[0]).ToNot(ContainSubstring(" "))
```

Comment:
Please do not make negative checks. You should check for the real output you expect. I know there a bunch of bad examples in the existing tests but we should do a better job for new tests.

If the numbers are not consistent you can use `To(MatchRegexp(...))` to match a regex instead.
Also just call `session.OutputToString()` to get the output as one string.

Reply:
No problem, I update the PR with an extra regex.